### PR TITLE
Reorganise cli E2E folder structure

### PR DESCRIPTION
**Pull request by praveen4g0**
# Changes

- No changes to code

- Restructured/Reorganise folders for better readability.

- Remove unused code

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Run the code checkers with `make check`
- [x] Regenerate the manpages, docs and go formatting with `make generated`
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
release-note
```

### CODE REVIEW

**Comment by praveen4g0** [comment]
@danielhelfand I have organised folder structure, please do let me know if you feel we can organise it  in better way.

File: test/assert/assert.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+// NilError fails the test immediately if err is not nil.
+func NilError(t *testing.T, err error, msgAndArgs ...interface{}) {
+	assert.NilError(t, err, msgAndArgs)
+}
+
+func Assert(t *testing.T, comparison interface{}, msgAndArgs ...interface{}) {
+	assert.Assert(t, comparison, msgAndArgs...)
+}
```
Comment (vdemeester) [inline test/assert/assert.go:36]: How much value do this add ? (aka the signature are almost the same, there is not extra "work", we could use what they do directly in the code :angel: )

Comment (praveen4g0) [reply test/assert/assert.go:36]: Agree @vdemeester I will remove this wrapper

**Comment by praveen4g0** [comment]
@pradeepitm12 @vdemeester can we merge this ?

**Pull request merged**